.github/workflows/rust.yml

```diff
@@ -43,18 +43,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-18.04, windows-2019, macos-10.15 ]
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
```

Comment:
It seems like a lot of this file was reformatted and that makes it a little harder to view the diffs.

Reply:
Ah, yeah, sorry about that. I'll revert those changes.

Reply:
Ah, one note: You can hide whitespace changes in diffs

![image](https://user-images.githubusercontent.com/3087225/148832008-732b9e34-e46f-4f0f-8da2-af1f13c8eb51.png)

I usually have that enabled which is probably why I didn't noticed the formatting diffs.